SECURITY.md

```diff
@@ -6,7 +6,7 @@ Please encrypt your message to us; please use our PGP key. The key fingerprint i
 
 F988 7BEA 027A 049F AE8E  5CAA D125 8932 BE24 C5CA
 
-The key is available from [pgp.mit.edu](https://pgp.mit.edu/pks/lookup?op=get&search=0xF9887BEA027A049FAE8E5CAAD1258932BE24C5CA) by searching for [grafana](https://pgp.mit.edu/pks/lookup?search=grafana&op=index).
+The key is available from [pgp.mit.edu](http://keys.gnupg.net/pks/lookup?search=0xF9887BEA027A049FAE8E5CAAD1258932BE24C5CA&fingerprint=on&op=index) by searching for [grafana](https://pgp.mit.edu/pks/lookup?search=grafana&op=index).
```

Comment:
Getting the keys via HTTP seems very bad and their HTTPS is broken for me 😢

Reply:
Fair.